### DISCUSSION TITLE
Does Dialogic support godot's built in translation system?

### DISCUSSION THREAD

**Discussion by yohannes-jo** [dialogic-godot/dialogic#1467] [Q&A]
I wanted to use both but i haven't seen any tutorials or discussions talking about it. I want to add my native language in the game as an option, what do I do? What are my options?

**Comment by Jowan-Spooner** [comment]
Hi, dialogic 1 supports translations by treating the input of a text event as a translation key. This is a setting you can enabled in the dialogic settings.

Dialogic 2 has a more complex solution that supports translation in csv files but not in the .po format. The system in 2.x is more flexible as events have unique translation identifiers. Check out the Dialogic>Settings>Translation tab in 2.x for more information. Keep in mind it's still in alpha.
